packages/ag-charts-website/src/features/api-documentation/components/JsObjectDetails.tsx

```diff
@@ -0,0 +1,39 @@
+import classnames from 'classnames';
+import styles from './ApiDocumentation.module.scss';
+import type { FunctionComponent, ReactNode } from 'react';
+import type { JsObjectSelection } from '../types';
+import { JsObjectPropertyView } from './JsObjectPropertyView';
+import type { Framework } from '@ag-grid-types';
+
+interface Props {
+    selection: JsObjectSelection;
+    framework: Framework;
+}
+
+function JsObjectDetailsContainer({ children }: { children: ReactNode }) {
+    return (
+        <div className={styles.apiReferenceOuter}>
+            <table className={classnames(styles.reference, styles.apiReference, 'no-zebra')}>
+                <colgroup>
```

Comment:
What's the purpose of the `<colgroup>` here. Is it an accessibility thing?

Reply:
It doesn't seem to be used for anything, so I've removed it.